src/wp-includes/class-wp-scripts.php

```diff
@@ -390,8 +425,20 @@ public function do_item( $handle, $group = false ) {
 			return true;
 		}
 
+		if ( '' !== $strategy ) {
+			$strategy = ' ' . $strategy;
+			if ( ! empty( $after_non_standalone_handle ) ) {
+				$strategy .= sprintf( " onload='wpLoadAfterScripts(%s)'", esc_attr( wp_json_encode( $handle ) ) );
```

Comment:
Let's use another variable name instead of overloading the variable that is intended just for the value of the strategy with some other purpose to include HTML, which can easily lead to mistakes.
```suggestion
			$strategy_attrs = '  ' . $strategy;
			if ( ! empty( $after_non_standalone_handle ) ) {
				$strategy_attrs .= sprintf( " onload='wpLoadAfterScripts(%s)'", esc_attr( wp_json_encode( $handle ) ) );
```

We'll also need to initialize the `$strategy_attrs` variable to an empty string outside of this if clause then and use that in the `sprintf` below.

Reply:
Note this is also addressed by https://github.com/10up/wordpress-develop/pull/58

Reply:
@westonruter I've approved https://github.com/10up/wordpress-develop/pull/58 (although in draft form) and left a comment RE tests, I think we're going to have to update test logic to use the script generation functions you use. I've asked whether you think that should be done in the same branch/PR or separate. Probably should be done in the same branch, happy to pick that up.

Reply:
@10upsimon I'd be grateful if you did pick that up. Thank you!

Reply:
@westonruter sure thing! I'll pick this up with pleasure.

Reply:
> Let's use another variable name instead of overloading the variable that is intended just for the value of the strategy with some other purpose to include HTML, which can easily lead to mistakes.

Alternatively, also addressed in https://github.com/10up/wordpress-develop/pull/62

Reply:
Noting that https://github.com/10up/wordpress-develop/pull/62 is now merged into this base PR